jwst/background/background_sub.py

```diff
@@ -81,10 +191,13 @@ def average_background(bkg_list, sigma, maxiters):
         The averaged background exposure
     """
 
-    # Determine the dimensionality of the background files
-    bkg_model = datamodels.open(bkg_list[0])
-    bkg_dim = len(bkg_model.data.shape)
-    image_shape = bkg_model.data.shape[-2:]
+    # Determine the dimensionality of the input file
+    in_model = datamodels.open(input_model)
+    bkg_dim = len(in_model.data.shape)
+    image_shape = in_model.data.shape[-2:]
+    in_model.close()
```

Comment:
I hope this doesn't have the indirect effect of closing the original `input_model` too.

Reply:
changed as above